package.json

```diff
@@ -9,13 +9,13 @@
     "@wordpress/dependency-extraction-webpack-plugin": "^3.4.1",
     "@wordpress/env": "^5.3.0",
     "10up-toolkit": "^4.3.1",
-    "cypress": "^9.5.2",
+    "cypress": "^11.2.0",
     "node-sass": "^7.0.3",
     "node-wp-i18n": "^1.2.6"
   },
   "scripts": {
-    "cypress:open": "cypress open --config-file tests/cypress/config.json",
-    "cypress:run": "cypress run --config-file tests/cypress/config.json",
+    "cypress:open": "cypress open --config-file tests/cypress/config.config.js",
+    "cypress:run": "cypress run --config-file tests/cypress/config.config.js",
```

Comment:
@jayedul can you confirm if the file name is correct?

Reply:
Yes, names are correct as long as the file exists in `./tests/cypress/` directory. Cypress 11+ moved to `.js` from `.json` for configuration.